### PR TITLE
fix(component): dont open select on focus

### PR DESCRIPTION
**Pull request by rtalvarez**
## What?

Don't open the select component when focus occurs

## Why?

It is not intended behavior, it's not native behavior, and nobody else does this.
It also creates some issues with modals, if the select is the first element it auto expands it causing weird UX

## Screenshots/Screen Recordings

https://user-images.githubusercontent.com/6025510/159347918-22b5b472-5de1-4be6-95aa-c74019bb2653.mov

## Testing/Proof

Verified locally

### CODE REVIEW

**Review by MariaJose — COMMENTED** [review]
Looks good 👍🏼 , just curious if we want to apply the same logic for the `multiSelect` component as well?

**Review by chanceaclark — APPROVED** [review]

**Review by MariaJose — APPROVED** [review]

**Review by jorgemoya — COMMENTED** [review]
Can you do the same for `MultiSelect`?

File: packages/big-design/src/components/MultiSelect/MultiSelect.tsx [hunk 1]
```diff
@@ -330,8 +330,6 @@ export const MultiSelect = typedMemo(
                 !isOpen && openMenu();
               },
               onFocus: (event) => {
-                !isOpen && openMenu();
-
```
Comment (MariaJose) [inline packages/big-design/src/components/MultiSelect/MultiSelect.tsx:334]: ❤️

Comment (rtalvarez) [reply packages/big-design/src/components/MultiSelect/MultiSelect.tsx:334]: Thanks for the comment i completely forgot `MultiSelect` was a thing HAHA 😅 😅

**Pull request merged**